00_ideas.md

```diff
@@ -150,4 +150,36 @@ To do so for this project, we ask you to try setting up and contributing to this
 bare-bones starting point for a project admin backend: [https://github.com/OpenHumans/oh-proj-management](https://github.com/OpenHumans/oh-proj-management)
 
 
+-----
+
+### Extending the Python module for the Open Humans API
+Many projects interact with Open Humans through our APIs (e.g. uploading/downloading/deleting data and sending messages). Many of these projects (including our own) are written in Python. We already have a basic Python library that enables the partial use of the full Open Humans API but it still lacks some of the API features that we want to see added.
+
+In addition the extended library should then also be used to create a re-usable `Django` web application that can easily be plugged in into larger Django web apps to make interfacing with Open Humans easier.
+
+Ultimately both the Python API client and the reusable web app should be published as `pip`-installable modules (with the `Django` app building on top of the general API library).
```

Comment:
"published in PyPI as"?

(I think PyPI should be mentioned here, calling things "pip-installable" is functionally good but also nice I think to explain where publication occurs)